### DISCUSSION TITLE
is llama.cpp now faster than exllamav2? i just ran a sample and it seems so. is this how it is now?

### DISCUSSION THREAD

**Discussion by sprappcom** [ggml-org/llama.cpp#5397] [Q&A]
llama.cpp now the fastest way to run llms?

**Comment by slaren** [comment]
Are you using flash attention with exllama?

**Reply by sprappcom** [reply]
nope.

do u know how to run the fastest llama ever? like the fastest way to run llama.
i've checked decillm, (i cant find a way to run together ai locally).

any ideas?

**Reply by slaren** [reply]
From what I understand, exllama is significantly faster when used with flash attention.

**Reply by sprappcom** [reply]
i tried exllama. it's faster by 10-15% not sure if flash attention is used.
